src/middlewares/errorLogger.ts

```diff
@@ -0,0 +1,11 @@
+import { NextFunction, Request, Response } from "express";
+
+export const errorLogger = (err: Error, req: Request, res: Response, next: NextFunction) => {
+  res.status(500).send({
```

Comment:
I believe Logger should not send 500 response. Logger should add error description in the log.
And somewhere should be error handler that will send responses with error status